enterprise/utils.py

```diff
@@ -399,6 +398,64 @@ def get_enterprise_customer_for_user(auth_user):
     return enterprise_customer_user.enterprise_customer
 
 
+def get_enterprise_customer_user(user_id, enterprise_uuid):
+    """
+    Return the object for EnterpriseCustomerUser.
+
+    Arguments:
+        user_id (str): user identifier
+        enterprise_uuid (UUID): Universally unique identifier for the enterprise customer.
+
+    Returns:
+        (EnterpriseCustomerUser): enterprise customer user record
+    """
+    try:
+        # pylint: disable=no-member
+        enterprise_customer_user = enterprise.models.EnterpriseCustomerUser.objects.get(
+            enterprise_customer__uuid=enterprise_uuid,
+            user_id=user_id
+        )
+    except enterprise.models.EnterpriseCustomerUser.DoesNotExist:
+        return None
+
+    return enterprise_customer_user
+
+
+def user_requires_data_sharing_consent(enterprise_customer_user, course_id=None):
+    """
+    Determine if user fulfills data sharing consent requirement for enterprise.
+
+    Arguments:
+        course_id (str): The ID of the course in which the learner was enrolled
+        enterprise_customer_user (EnterpriseCustomerUser): enterprise customer
+            user record
+
+    Returns:
+        Boolean: enterprise customer user required to provide data sharing
+            consent.
+
```

Comment:
At edX we write docstrings mostly in `Google Style` manner and I added this new line deliberately. You can find multiline docstring examples here: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings